hs_core/management/utils.py

```diff
@@ -28,6 +28,32 @@
 import logging
 
 
+def migrate_core_meta_elements(orig_meta_obj, comp_res):
```

Comment:
This is duplicate code and likely will cause code conflict. Since the netcdf migration branch has been merged this branch can now be updated with develop to remove this duplicate code. I can do that tonight.

Reply:
Looks like there is no code conflict reported by Jenkins. So ignore my comments above.